test/02_elementwise/elementwise_binary_op_test.cpp

```diff
@@ -568,20 +566,23 @@ namespace hiptensor
             } // if (testOptions->performValidation())
 
             CHECK_HIPTENSOR_ERROR(hiptensorDestroy(handle));
+            CHECK_HIPTENSOR_ERROR(hiptensorDestroyPlanPreference(planPref));
```

Comment:
Good catch. These are because of the 2.0 API changes

Reply:
Reverted.